dll/win32/uxtheme/system.c

```diff
@@ -545,12 +545,21 @@ static HRESULT UXTHEME_ApplyTheme(PTHEME_FILE tf)
 		(lstrlenW(tf->pszSelectedSize)+1)*sizeof(WCHAR));
             RegSetValueExW(hKey, szDllName, 0, REG_SZ, (const BYTE*)tf->szThemeFile, 
 		(lstrlenW(tf->szThemeFile)+1)*sizeof(WCHAR));
+#ifdef __REACTOS__
+        {
+            WCHAR buf[100];
```

Comment:
```suggestion
            WCHAR buf[sizeof("4294967295")];
```
or
```suggestion
            WCHAR buf[11];
```
is more than enough :D (the LANGID is a 32-bit value)